collections/Auct_F/MS_Auct_F_2_8.xml

```diff
@@ -37,49 +36,328 @@
+                        <finalRubric>Explicit liber georgicorum ·iiij·</finalRubric>
+                        <bibl type="repertory"><ref target="http://webserver.erwin-rauner.de/php_ancarm/ancarm_refwerke.php?opus=1001&amp;var=ICL&amp;nr_char=12694"><title>ICL</title> 12694</ref></bibl>
+                     </msItem>
                   </msItem>
-                  <msItem n="3" xml:id="MS_Auct_F_2_8-item3">
+                  <msItem>
+                     <locus>(fol. 53v)</locus>
+                     <author key="person_285292080">Pseudo-Ovid</author>
+                     <title key="work_11463">De xii libris Aeneidos</title>
+                     <rubric>Versiculi ouidii nassonis super eneidos</rubric>
+                     <incipit>Primus habet lybicam ueniant ut troes in urbem</incipit>
+                     <explicit>Ultimus inponit bello turni nece finem.</explicit>
+                     <bibl type="repertory"><ref target="http://webserver.erwin-rauner.de/php_ancarm/ancarm_refwerke.php?opus=1001&amp;var=ICL&amp;nr_char=12542"><title>ICL</title> 12542</ref></bibl>
+                     <bibl type="repertory"><ref target="http://webserver.erwin-rauner.de/php_ancarm/ancarm_refwerke.php?opus=1000&amp;var=WIC&amp;nr_char=14661"><title>WIC</title> 14661</ref></bibl>
+                     <note><title>Anthologia Latina</title> 634</note>
+                  </msItem>
+                  <msItem>
+                     <locus>(fols. 54r–226v)</locus>
                      <author key="person_8194433">Virgil</author>
                      <title key="work_4920">Aeneid</title>
-                     <note>(with gloss and commentary)</note>
-                     <textLang mainLang="la">Latin</textLang>
+                     <msItem>
+                        <locus>(fols. 54r–67r)</locus>
+                        <bibl>1.1–756</bibl>
+                        <rubric>Incipit aeneidorum liber primus</rubric>
+                        <incipit>Arma uirumque cano troiae qui primus ab oris</incipit>
+                        <explicit>Omnibus errantem terris et fluctibus aestas</explicit>
+                        <finalRubric>Explicit liber primus aeneidorum.</finalRubric>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 67r)</locus>
+                        <author key="person_285292080">Pseudo-Ovid</author>
+                        <title type="desc">Argument to <title>Aeneid</title> 2</title>
+                        <rubric>Incipit praefatio libri ·ii·</rubric>
+                        <incipit><supplied>F</supplied>unera dardanię narrat flecteque secundo.</incipit>
+                        <explicit defective="true">Somnum quod minitus acceperat haectoris atrum.</explicit>
+                        <bibl type="repertory"><ref target="http://webserver.erwin-rauner.de/php_ancarm/ancarm_refwerke.php?opus=1001&amp;var=ICL&amp;nr_char=2699"><title>ICL</title> 2699</ref></bibl>
```

Comment:
these links don't currently work...

Reply:
That is likely due to the `&amp;` URL encoding: <http://webserver.erwin-rauner.de/php_ancarm/ancarm_refwerke.php?opus=1001&var=ICL&nr_char=2699> works for me right now.